### PR TITLE
WEBUI-1171: fixed direct download URL failing with appended clientRea…

### PR DESCRIPTION
**Pull request by swarnadipa-dev**
…son parameter

### CODE REVIEW

**Review by Nishant0928 — DISMISSED** [review]

**Review by rakeshkumar1019 — DISMISSED** [review]

File: elements/document/nuxeo-picture-formats.js [hunk 1]
```diff
@@ -90,8 +90,16 @@ Polymer({
...
             downloadUrl: view.content.downloadUrl,
+            data: view.content.data,
           };
         })
       : [];
   },
+
+  _getDownloadUrl(item) {
+    if (item) {
+      return item.downloadUrl ? item.downloadUrl : item.data;
```
Comment (rajatgoel-gl) [inline elements/document/nuxeo-picture-formats.js:101]: Check if Nullish Coalescing Operator can simplfy the code

File: elements/nuxeo-document-blob/nuxeo-document-blob.js [hunk 2]
```diff
@@ -120,4 +120,11 @@ Polymer({
     }
     return obj;
   },
+
+  _getDocumentBlobUrl() {
+    if (this.blob) {
+      return this.blob && this.blob.downloadUrl ? this.blob.downloadUrl : this.blob.data;
```
Comment (rajatgoel-gl) [inline elements/nuxeo-document-blob/nuxeo-document-blob.js:126]: Check if Nullish Coalescing Operator can simplfy the code

**Review by rajatgoel-gl — CHANGES_REQUESTED** [review]

File: elements/document/nuxeo-picture-formats.js (hunk 1 above)
Comment (swarnadipa-dev) [reply elements/document/nuxeo-picture-formats.js:101]: done

File: elements/nuxeo-document-blob/nuxeo-document-blob.js (hunk 2 above)
Comment (swarnadipa-dev) [reply elements/nuxeo-document-blob/nuxeo-document-blob.js:126]: done

**Review by rajatgoel-gl — APPROVED** [review]

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by Nishant0928 — APPROVED** [review]

**Pull request merged**